pkg/services/featuremgmt/registry.go

```diff
@@ -715,5 +715,12 @@ var (
 			FrontendOnly: true,
 			Owner:        grafanaPluginsPlatformSquad,
 		},
+		{
+			Name:         "dashgpt",
+			Description:  "Enable AI powered features",
```

Comment:
Driveby comment but this is quite general, we may end up with other AI powered features in future - could it be more specific to dashgpt? 🙏

Reply:
Hi @sd2k - that is a great point.

Other than updating feature flag description to more specific to dashboard related tools, another approach could be making the feature flag itself more general (i.e. `genai` vs `dashgpt`) or something to it can be used for other AI features as well 🤷 what are your thoughts around this?

Reply:
Both seem sensible to me really, I'd defer to whatever the standard practise is for Grafana feature flags (i.e. granular vs coarse).

Selfishly, I do think it would be nice to have a `genai` feature flag we could use for other things I'd like to add 😁

Reply:
Discussed this the dashgpt working group and we decided to keep the `dashgpt` specific naming so that the feature flag can be only tied to dashgpt related features (which simplifies management of feature flag and its life cycle)